spec/rubocop/cop/lint/script_permission_spec.rb

```diff
@@ -14,10 +14,16 @@
     let(:execution) { false }
 
     it 'registers an offense for script permission' do
-      expect_offense(<<-RUBY.strip_indent)
-        #!/usr/bin/ruby
-        ^^^^^^^^^^^^^^^ Script file example.rb doesn't have execute permission.
-      RUBY
+      if RuboCop::Platform.windows?
```

Comment:
Mmm, the patch is intentional, I made this cop a noop in Windows because it has no notion of shebangs or executable bits, so it seems to me that it just doesn't apply to Windows. Which typo do you mean?